### DISCUSSION TITLE
谢谢Mike，续费问题已经解决！但是，说好的一个月优惠呢？

### DISCUSSION THREAD

**Discussion by kenleves** [getlantern/lantern-forum-cn#925] [General]
首先谢谢@Mike777Khoo
续费问题已经解决了！
另外跟各位兄弟说一下，续费问题提交后，请耐心等待。
我是9月28日提交的问题，大约在10月2日续费成功，用时较长，不要急。
蓝灯我用了大概7年了，一直很稳，请放心食用。
当然，当年蓝灯336买两年，现在337买一年……

好了，其次，言归正传。
Mike啊，虽然续费问题解决了，但是，说好续费一年，然后加一个月时长呢？
我从2024年10月15日续费一年，加一个月优惠，应该是2025年11月15日……
So，然后呢……
继续努力MIke，我看好你哦！

<img width="947" alt="2024年10月到期" src="https://github.com/user-attachments/assets/918c1a79-4799-4bad-a438-e09698aae57e">
<img width="790" alt="蓝灯续期" src="https://github.com/user-attachments/assets/45df4936-3392-4ee5-be3d-5005b2a1afc8">
<img width="423" alt="只有一年" src="https://github.com/user-attachments/assets/bfce604f-1a56-490b-a93f-2a6af77c6c03">
<img width="294" alt="2017年蓝灯" src="https://github.com/user-attachments/assets/390a0474-fe0b-43c8-94d5-f444bb68b32a">

**Comment by MaxMKMax** [comment]
@kenleves 您好. 好的, 我再次帮您和技术人员申请那一个月. 估计他们漏掉了. 不好意思啊. 或者系统出错了. 
谢谢您支持蓝等这么多年. 也谢谢您夸奖我. 感谢您的赞美! 

祝您假期愉快和周末愉快!

**Comment by MaxMKMax** [comment]
@kenleves 您好. 我们已经给您添加 3 个月了. 多送您 2 个月来报答您的赞美!!!

请您也给我们 5 星好评. 在哪个app商店也行. 谢谢啦!

**Reply by kenleves** [reply]
谢谢@Mike777Khoo，真的很高兴，并不仅仅是因为增加了3个月的使用时长，而是……某种信任。
毕竟，默默用了蓝灯这么久，其实，我也很惊讶自己，能用一个产品（时不时抽风的产品）用这么久的时间。
比如开大会的时候、比如某些奇妙的时间点等等……
让我们一起努力吧，希望世界会因为一些小小的执拗与坚持，变得越来越好，你说是吧MIke！

**Reply by MaxMKMax** [reply]
@kenleves 您好. 请问怎么称呼您? 我们也想谢谢您支持蓝等这么久. 一言既出,驷马难追! 我们说到做得到的! 非常同意你说的话, 一起努力! 希望这世界变得越来越好. 也希望网络可以拉近两国的距离.

 我们也非常乐意和理性沟通的用户解决问题和建立更良好的关系.

非常高兴看到您很开心. 这样我也有动力起来了. 您对我的信任让我更加有动力. 


-Mike 邱 (姜子牙姓氏的后代)

**Comment by MaxMKMax** [comment]
@kenleves 以后有什么需要帮忙的请联系我. 周末愉快!
